build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val root = (project in file("."))
       "org.scalatest" %% "scalatest" % "3.0.5" % "it,test",
       "junit" % "junit" % "4.13.2" % "it,test",
       "org.testcontainers" % "mysql" % "1.15.1" % "it,test",
-      "com.zaxxer" % "HikariCP" % "3.4.5"
```

Comment:
Is HikariCP not used?

Reply:
Nevermind looks like all uses were deleted.

Reply:
yes I added this library initially for Junit. But now without this library also we are able to get the connection. So removed this.

Reply:
yes I added this library initially for Junit. But now without this library also we are able to get the connection. So removed this.